layout/columns2.php

```diff
@@ -22,12 +22,24 @@
 
 defined('MOODLE_INTERNAL') || die();
 
+// Get Theme Nightingale's JS variables
+$theme_nightingale_variables = array();
+
+if(user_has_role_assignment($USER->id,5)) { // 5 is the role id for Student
```

Comment:
What does this get used for?

Reply:
There is a "Separate Groups" dropdown on Admin side and changes to "Text" for Participants. Since participants will mostly always belong to single Group and the group they belong is also listed on forum listing page, @Pro-Paul asked to remove this repetition. 

This is to identify if the logged in user is "Student" role (ID:5) to accordingly hide group selector text.